macOS/geotoad.rb

```diff
@@ -0,0 +1,21 @@
+class Geotoad < Formula
+  desc "Query tool to query the geocaching.com website"
+  homepage "https://github.com/steve8x8/geotoad"
+  url "https://github.com/steve8x8/geotoad/archive/3.25.4.tar.gz"
```

Comment:
You manually update this `url` and the below `sha256` (`shasum --algorithm 256` on the archive) and its done. It’s only updated for whatever version *you* want.